app/assets/scripts/views/country/country.js

```diff
@@ -0,0 +1,279 @@
+import React, { useState, useEffect } from 'react';
+import { PropTypes as T } from 'prop-types';
+import { connect } from 'react-redux';
+import _ from 'lodash';
+
+import { formatThousands } from '../../utils/format';
+import config from '../../config';
+import { openDownloadModal } from '../../actions/action-creators';
+
+import Header, { LoadingHeader, ErrorHeader } from '../../components/header';
+import LoadingMessage from '../../components/loading-message';
+import InfoMessage from '../../components/info-message';
+import LocationCard from '../../components/location-card';
+import CountryMap from './map';
+
+const defaultLocations = {
+  locationFetched: false,
+  locationFetching: false,
+  locationError: null,
+  locations: null,
+};
+
+const defaultCountry = {
+  countryFetched: false,
+  countryFetching: false,
+  countryError: null,
+  country: null,
+};
+
+function Country(props) {
+  const id = props.match.params.name;
+  const [
+    { locationFetched, locationFetching, locationError, locations },
+    setLocations,
+  ] = useState(defaultLocations);
+  const [
+    { countryFetched, countryFetching, countryError, country },
+    setCountry,
+  ] = useState(defaultCountry);
+
+  useEffect(() => {
+    const fetchLocations = id => {
+      setLocations(state => ({
+        ...state,
+        locationFetching: true,
+        locationError: null,
+      }));
+      let limit = 10000;
+      fetch(`${config.api}/locations?limit=${limit}&country=${id}`)
+        .then(response => {
+          if (response.status >= 400) {
+            throw new Error('Bad response');
+          }
+          return response.json();
+        })
+        .then(
+          json => {
+            setLocations(state => ({
+              ...state,
+              fetched: true,
+              locationFetching: false,
+              locations: json.results,
+            }));
+          },
+          e => {
+            console.log('e', e);
+            setLocations(state => ({
+              ...state,
+              locationFetched: true,
+              locationFetching: false,
+              locationError: e,
+            }));
+          }
+        );
+    };
+
+    const fetchCountry = id => {
+      setCountry(state => ({
+        ...state,
+        countryFetching: true,
+        countryError: null,
+      }));
+      let limit = 1;
+      fetch(`${config.api}/countries?limit=${limit}&country=${id}`)
```

Comment:
Maybe we should change the /countries endpoint to accept `​/countries​/{country_id}`, for consistency with /locations and /projects. What do you think, @bitner ?